_episodes/33.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: episode
+title:  "33: Trino becomes highly available for high demand"
+date: 2022-02-17
+tags: architecture high-availability
+youtube_id: ""
+wistia_id: ""
+sections: 
+   - title: "Concept of the month"
+     desc: "High Availability with Trino"
+     time: 0
+   - title: "PR of the month"
+     desc: "PR 8956 Add support for external db for schema management in mongodb connector"
+     time: 0
+   - title: "Bonus PR of the month"
+     desc: "PR 8202 Metadata for alias in elasticsearch connector only uses the first mapping"
+     time: 0
+   - title: "Demo of the month"
+     desc: "Trino Fiddle: A tool for easy online testing and sharing of Trino SQL problems and their solutions"
+     time: 0
+   - title: "Question of the month"
+     desc: "Does trino hive connector supports CarbonData?"
+     time: 0
+---
+
+## Guests
+
+ * Ramesh Bhanan, Vice President, at [Goldman Sachs](https://developer.gs.com/discover/home)
+  ([@ramesh-bhanan-byndoor](https://www.linkedin.com/in/ramesh-bhanan-byndoor/)).
+ * Sambit Dikshit, Managing Director, Tech Fellow at [Goldman Sachs](https://developer.gs.com/discover/home)
+  ([@sambitdixit](https://www.linkedin.com/in/sambitdixit/)).
+ * Siddhant Chadha, Senior Data Engineer at [Goldman Sachs](https://developer.gs.com/discover/home)
+ ([@siddhant-chadha](https://www.linkedin.com/in/siddhant-chadha-838136142/)).
+ * Suman Baliganahalli Narayan Murthy, Vice President at [Goldman Sachs](https://developer.gs.com/discover/home)
+ ([@suman-b-n](https://www.linkedin.com/in/suman-b-n-08-03-1990/)).
+ * Sumit Halder, Vice President at [Goldman Sachs](https://developer.gs.com/discover/home)
+  ([@sumit-halder](https://www.linkedin.com/in/sumit-halder-a3732482/)).
+ 
+## Releases 369, 370, and 371
+
+Trino 369
+
+* Experimental support for task level retries.
+* Support for groups in OAuth2 claims.
+* Column comments in ClickHouse connector.
+* Write Bloom filters in ORC files.
+* Procedure for optimizing Iceberg tables.
+
+Trino 370
+
+* Add CLI support for ARM64
+* Improved performance for ORC
+* Improved performance for map and row types
+* Reduced latency for OAuth2.0 authentication
+
+Trino 371
+
+tbd
```

Comment:
* Support for secrets and user group selector in resource group manager.
* Support AWS role session name in S3 security mapping configuration.
* Many bug fixes.